pkgs/by-name/al/alt-tab-macos/package.nix

```diff
@@ -1,16 +1,18 @@
-{ lib
-, stdenvNoCC
-, fetchurl
-, unzip
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  gitUpdater,
+  unzip,
 }:
```

Comment:
Revert this, the previous style is how it's normally done in nixpkgs.

Reply:
Shouldn't we be complying with https://github.com/nixos/rfcs/pull/166 ?

Reply:
That works for me, I thought the attr format had remained the same, my bad :)